docs/en/02_Developer_Guides/01_Templates/11_Partial_Template_Caching.md

```diff
@@ -0,0 +1,282 @@
+---
+title: Partial Template Caching
+summary: Cache a section of a template Reduce rendering time with cached templates and understand the limitations of the ViewableData object caching.
+icon: tags
+---
+
+## Partial template caching
+
+Partial template caching is a feature that allows caching of rendered portions of templates. Cached content
+is fetched from a [cache backend](../performance/caching), instead of being regenerated repeatedly.
+
+
+### Base syntax
+
+Here is an example of the basic usage. This is not a definitive example of the syntax, but it shows the most
+common use case.
+
+[note]
+See also [Complete Syntax definition](#complete-syntax-defintition) section
+[/note]
+
+```ss
+<% cached $CacheKey if $Condition %>
```

Comment:
I don't think `if $Condition` is common.  I can't recall ever seeing it in a project.  See comments about `$IsCacheable` above

Reply:
> I don't think if $Condition is common.

This should be common, and that should be our default recommendation. Unless user knows exactly what they're doing, they should be using conditions. Problems with missing conditions are explained below (see the note "Although `$Condition` is optional, it is strongly recommended.").

Reply:
So, what would be an example of a condition?  Presumably it would be a function on a Page_Controller or a DataObject?

Reply:
sure, here we go:

```php
public function getCacheKey() { return $this->ID; }
public function getCondition() { return $this->ID > 0; }
```

Reply:
cool, though they need to be in the docs not a github comment =)